packages/core/admin/ee/admin/pages/SettingsPage/pages/ReviewWorkflows/hooks/tests/useReviewWorkflows.test.js

```diff
@@ -29,7 +31,15 @@ const ComponentFixture = ({ children }) => (
 function setup(id) {
   return new Promise((resolve) => {
     act(() => {
-      resolve(renderHook(() => useReviewWorkflows(id), { wrapper: ComponentFixture }));
+      resolve(
+        renderHook(
+          () => ({
+            ...useReviewWorkflows(id),
+            queryClient: useQueryClient(),
```

Comment:
Might need a comment why you're passing `useQueryClient` here? 🤔

Reply:
Good catch. That was a leftover which is not needed anymore. Removed in af7c2f76e8a7cf020923273f8efc8a768cc440bc